docs/organizations/managing-security-and-risk.md

```diff
@@ -0,0 +1,115 @@
+# Managing security and risk
+
+<!-- TODO remove this admonition before publishing. This is caused by https://codacy.atlassian.net/browse/HRZ-151 -->
+!!! important "This is a preview version of the Codacy documentation"
+    For the latest updates and improvements, [see the latest Cloud documentation](https://docs.codacy.com/) instead.
```

Comment:
I already the idea of adding an automatic banner for the pull request previews for some time now, but when I saw this change I decided to spend some minutes implementing it:

- https://github.com/codacy/docs/pull/1721
- https://github.com/codacy/codacy-mkdocs-material/pull/82